helm/robusta/values.yaml

```diff
@@ -1,6 +1,9 @@
 # most of this file is documented at https://docs.robusta.dev/master/user-guide/configuration.html
 # if you have any questions, feel free to ask via github issues or Slack (see link on robusta.dev)
 
+# NOTE: Do not use this file to install Robusta. Check the Installation isntructions to generate your generated_values.yaml(Helm values file) using RobustaCLI. 
```

Comment:
>  NOTE: Do not use this file to install Robusta.
Why not?

For the second half of the sentence, I would just say "see the installation instructions instead." No need to mention the generated_values or Robusta cli here IMO. They can read about it in the installation page.

Also, typo in `isntructions`.

Reply:
Fixed